packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -152,28 +152,28 @@ class _CpuProfilerState extends State<CpuProfiler>
     final theme = Theme.of(context);
     final textTheme = theme.textTheme;
     final colorScheme = theme.colorScheme;
-    final currentTab =
-        widget.tabs.isNotEmpty ? widget.tabs[_tabController.index] : null;
-    final hasData =
-        data != CpuProfilerController.baseStateCpuProfileData && !data.isEmpty;
-
+    if (widget.tabs.isEmpty) {
+      return Container();
+    }
+    final currentTab = widget.tabs[_tabController.index];
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: <Widget>[
-        SizedBox(
-          height: defaultButtonHeight,
-          child: Row(
-            children: [
-              TabBar(
-                labelColor:
-                    textTheme.bodyText1?.color ?? colorScheme.defaultForeground,
-                isScrollable: true,
-                controller: _tabController,
-                tabs: widget.tabs,
-              ),
-              const Spacer(),
-              if (hasData) ...[
-                if (currentTab!.key != CpuProfiler.summaryTab) ...[
+        AreaPaneHeader(
+          needsTopBorder: false,
+          leftPadding: 0,
+          tall: true,
+          title: TabBar(
+            labelColor:
+                textTheme.bodyText1?.color ?? colorScheme.defaultForeground,
+            isScrollable: true,
+            controller: _tabController,
+            tabs: widget.tabs,
+          ),
+          actions: [
+            Row(
```

Comment:
do we need this wrapper row?

Reply:
We did for some reason before, but I must have massaged things enough to make it unnecessary now. Removed.